common/button/button-en.html

```diff
@@ -0,0 +1,415 @@
+
+<h4>Working example</h4>
+<h5>Default:</h5>
+<button type="submit">Button</button>
+<input type="button" value="Input" />
+<input type="submit" value="Submit" />
+<a href="#" >Link</a>
+
+<h5>Styled:</h5>
+<button class="btn btn-default" type="submit">Button</button>
+<input class="btn btn-default" type="button" value="Input" />
+<input class="btn btn-default" type="submit" value="Submit" />
+<a class="btn btn-default" href="#" role="button">Link</a>
+
+<h4>Code Sample</h4>
+<pre><code>
+	<span class="com">// Default button:</span>
+	&lt;button type="submit"&gt;
+	Button
+	&lt;/button&gt;
+	&lt;input type="button" value="Input"/&gt;
+	&lt;input type="submit" value="Submit"/&gt;
+	&lt;a href="#" role="button"&gt;
+	Link
+	&lt;/a&gt;
+
+	<span class="com">// Styled button:</span>
+	&lt;button class="<strong>btn btn-default</strong>" type="submit"&gt;
+	Button
+	&lt;/button&gt;
+	&lt;input class="<strong>btn btn-default</strong>" type="button" value="Input"/&gt;
+	&lt;input class="<strong>btn btn-default</strong>" type="submit" value="Submit"/&gt;
+	&lt;a class="<strong>btn btn-default</strong>" href="#" role="button"&gt;
+	Link
+	&lt;/a&gt;
+</code></pre>
+
+<h2 id="enhanced">Enhanced use</h2>
+
+<h3 id="colours/styles">Colours/Styles</h3>
+<p>Use to add colours to buttons. Colours signify various things such as a link to warning information (caution button) to a link to the next learning module (success button).</p>
+<h4>Working example</h4>
+<button type="button" class="btn btn-default">Default</button>
+<button type="button" class="btn btn-primary">Primary</button>
+<button type="button" class="btn btn-success">Success</button>
+<button type="button" class="btn btn-info">Info</button>
+<button type="button" class="btn btn-warning">Warning</button>
+<button type="button" class="btn btn-danger">Danger</button>
+<button type="button" class="btn btn-link">Link</button>
+
+<h4>Code sample</h4>
+<pre><code>
+	<span class="com">// Default:</span>
+	&lt;button type="button" class="btn <strong>btn-default</strong>"&gt;
+	Default
+	&lt;/button&gt;
+
+	<span class="com">// Primary:</span>
+	&lt;button type="button" class="btn <strong>btn-primary</strong>"&gt;
+	Primary
+	&lt;/button&gt;
+
+	<span class="com">// Success:</span>
+	&lt;button type="button" class="btn <strong>btn-success</strong>"&gt;
+	Success
+	&lt;/button&gt;
+
+	<span class="com">// Info:</span>
+	&lt;button type="button" class="btn <strong>btn-info</strong>"&gt;
+	Info
+	&lt;/button&gt;
+
+	<span class="com">// Warning:</span>
+	&lt;button type="button" class="btn <strong>btn-warning</strong>"&gt;
+	Warning
+	&lt;/button&gt;
+
+	<span class="com">// Danger:</span>
+	&lt;button type="button" class="btn <strong>btn-danger</strong>"&gt;
+	Danger
+	&lt;/button&gt;
+
+	<span class="com">// Link:</span>
+	&lt;button type="button" class="btn <strong>btn-link</strong>"&gt;
+	Link
+	&lt;/button&gt;
+	</code></pre>
+
+<h3 id="calltoaction">Call to Action</h3>
```

Comment:
Let add the name used in the design system.

```suggestion
<h3 id="calltoaction">Supertask (Call to Action)</h3>
```

Reply:
Addressed in commit  "Button: Addressed Review Comments"